integrators/methods.hpp

```diff
@@ -460,6 +471,20 @@ struct DormandالمكاوىPrince1986RKN434FM :
       {13.0 /   21.0, -20.0 /  27.0, 275.0 / 189.0, -1.0 /  3.0}}};
 };
 
+struct HeunEuler : EmbeddedExplicitRungeKutta {
```

Comment:
Let us remove this one (alternatively, add it to the convergence tests, and dig up a proper reference; but I can hardly be bothered, and if we want to use this one, something has likely gone terribly wrong.